app/views/requests/_needed_date.html.erb

```diff
@@ -3,8 +3,9 @@
     <%= f.label :needed_date, class: "#{label_column_class} required control-label" %>
     <div class='<%= content_column_class %>'>
       <%= f.date_field_without_bootstrap :needed_date, class: 'form-control', required: true, min: Time.zone.today, data: { request_type: current_request.type.underscore } %>
+      <div id='needed-date-warning' class='has-error needed-date-help-block'></div>
```

Comment:
should this be inside of an if clause so we only get the div when the needed date is bad?